crud/common/schema.lua

```diff
@@ -0,0 +1,107 @@
+local fiber = require('fiber')
+local msgpack = require('msgpack')
+local digest = require('digest')
+local vshard = require('vshard')
+
+local const = require('crud.common.const')
+
+local schema = {}
+
+local function call_reload_schema(replicasets)
+    for _, replicaset in pairs(replicasets) do
+        local master = replicaset.master
+        master.conn:reload_schema()
+    end
+end
+
+local reload_in_progress = false
+local reload_schema_cond = fiber.cond()
+
+local function reload_schema(replicasets)
+    if reload_in_progress then
+        reload_schema_cond:wait()
```

Comment:
It would be great to add some timeout here. It's easy to get deadlock here if something occasionally will throw or reload will hang for some reasons (e.g. network problem)